src/templates/home-page.js

```diff
@@ -103,7 +104,18 @@ export const HomePageTemplate = ({
             </div>
           </Spotlight>
         </section>
-        <Resources />
+        <section className="latest-news col-sm-12 col-md-12 col-lg-10">
```

Comment:
Is it intentional that you're using `latest-news` rather than `resources` for the class?